griptape/drivers/prompt_model/bedrock_jurassic_prompt_model_driver.py

```diff
@@ -14,6 +14,11 @@ class BedrockJurassicPromptModelDriver(BasePromptModelDriver):
     _tokenizer: BedrockJurassicTokenizer = field(default=None, kw_only=True)
     prompt_driver: Optional[AmazonBedrockPromptDriver] = field(default=None, kw_only=True)
 
+    @prompt_driver.validator
+    def validate_prompt_driver(self, _, prompt_driver):
+        if prompt_driver and prompt_driver.stream:
```

Comment:
Should we add a `supports_streaming` property to prompt model drivers and validate at the Bedrock prompt driver level? This way this pattern can be easily extended to HF and SM prompt/model drivers in the future.